build_tag_push.py

```diff
@@ -1,101 +1,130 @@
+	''' 
+	This function is using the yaml built in to parse the file which can be done by searching for 
+	the key in the dictionary for the yaml file. 
+
+	Example: name = dict_of_contents['name']
+	'''
 	with open(filename, "r") as file_contents:
 		dict_of_contents = yaml.load(file_contents) # might be sys.argv[1] with the GitHub URL to the yml file as the argument when running the python scrip
-	name = dict_of_contents['name']
-	#print name
-	build_script = dict_of_contents['build']
-	#print build_script
-	test_call = dict_of_contents['test']
-	#print test_call
-	docker_context_build = dict_of_contents['docker_context'][0]
-	#print docker_context_build
-	docker_context_static = dict_of_contents['docker_context'][1]
-	#print docker_context_static
-	return name, build_script, test_call, docker_context_build, docker_context_static # Returns a list of the values from the dictionary dict_of_contents
-
-# Step 2 - Builds and test the app according to that config file = NOT COMPLETE.
-# There needs to be a Build File .travis.yml on their github
-# Seems as though the GitHub repo needs to be added to the Travis CI account and then when there is a new push it will
-# run the tests in the .travis.yml file that is in that repo. Ask for clarification...
-#
-#
-#
-#
-#
-#
-#
-#
-#
-#
-
-# Step 3 - Runs docker build with the right context = PARTIALLY COMPLETE.
-# They will have their own Dockerfile in their GitHub repo (I am assuming) and this file is a text file with 
-# instructions to build image. Dockerfile -> docker build -> Docker Image, it is basically an automation of Docker Image
-# Creation. In the DockerFile, you generally do: FROM = tells Docker which image you base your image on, 
+	# name = dict_of_contents['name']
+	# build_script = dict_of_contents['build']
+	# test_call = dict_of_contents['test']
+	# docker_context_build = dict_of_contents['docker_context'][0]
+	# docker_context_static = dict_of_contents['docker_context'][1]
+	return dict_of_contents
+
+# Step 2 - Builds and test the app according to that config file = PENDING.
+
+# Is this something other than working with Travis CI or is this part of the next step? 
+# I was under the impression that Travis CI is what we are using to build and test their app. 
+# If not, can you elaborate more on what you want this step to be?
+
+def createContextDirectory(contextArray):
+	original_dir_path = os.getcwd() # Getting this path to copy over files later in this function. I am assuming
+									# this is where the original files were put that we will be working out of.
+
+	dir_name = "MVPStudio_temp_dir" 
+	# Should this be user input so it can be different names per group? -->
+	# dir_name = input('Enter a directory name for your Docker Context: ')
+
+	os.makedirs(dir_name, exist_ok=True) # Making the directory and saying to not raise an error if the directory already exists.
+
+	dir_path = original_dir_path+'/'+dir_name # This is providing the path to the Docker Context directory.
```

Comment:
use `os.path.join` to join paths. That handles this like joining `foo/` and `/bar` so you get `foo/bar` instead of `foo//bar`. However, in this case I think we want to be using a temp directory for their docker context.